index.html

```diff
@@ -62,9 +59,148 @@ <h2>LET’S CONNECT</h2>
+      </div>
+      <div class="footerbox">
+        <div class="let">
+          <h2 class="lets">LET’S CONNECT</h2>
+        </div>
+      </div>
+      <section class="social">
+        <ul class="social-icons-set">
+          <li class="social-icon">
+            <a href="#"><i class="fa-brands fa-github" target="_blank" aria-label="GitHub"></i></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"><i class="fa-brands fa-linkedin-in" target="_blank" aria-label="Linkedin"></i></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"><i class="fa-brands fa-angellist" target="_blank" aria-label="Angellist"></i></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"><i class="fa-brands fa-twitter" target="_blank" aria-label="Twitter"></i></a>
+          </li>
+          <li class="social-icon"><a href="#"><i class="fa-solid fa-m" target="_blank"
+                aria-label="Medium"></i></a>
+          </li>
+        </ul>
+      </section>
+      <div class="box3">
+        <p class="box3c">Get my resume</p>
+      </div>
+      <div class="skills">
+        <div class="frame1"></div>
+        <div class="frame2">
+          <div class="framel">
+            <h2 class="lan">
+              Languages
+            </h2>
+          </div>
+          <div class="framer">
+            <li class="soc"><a class="soc1" href="#"><i class="fa-solid fa-chevron-up fa-rotate-180"></i></a>
+            </li>
+          </div>
+        </div>
+      </div>
+      <ul class="list">
+        <li class="jsfr">
+          <div class="jst">
+            <p class="jste">JavaScript</p>
+            <img class="im" src="./images/Ellipse 1.png" alt="">
+          </div>
+        </li>
+        <li class="jsfr">
+          <div class="jst">
+            <p class="jste">HTML</p>
+            <img class="im" src="./images/Ellipse 1 (1).png" alt="">
+          </div>
+        </li>
+        <li class="jsfr">
+          <div class="jst">
+            <p class="jste">CSS</p>
+            <img class="im" src="./images/Ellipse 1 (2).png" alt="">
+          </div>
+        </li>
+      </ul>
+      
+        <div class="endtop">
+          <div class="endl">
+            <p class="framworks">Framworks</p>
+          </div>
+          <div class="endr">  <li class="soc"><a class="soc1" href="#"><i class="fa-solid fa-chevron-up fa-rotate-90"></i></a>
+          </li></div>
+        </div>
+        <hr class="w">
+        <div class="endtop">
+          <div class="endl">
+            <p class="framworks">Skills</p>
+          </div>
+          <div class="endr">  <li class="soc"><a class="soc1" href="#"><i class="fa-solid fa-chevron-up fa-rotate-90"></i></a>
+          </li></div>
+        </div>
+        <hr class="w">
```

Comment:
- [ ] As the previous reviewer suggested, Kindly work on it. :+1: 

> * [ ]  Please decrease the `font-weight` of the technologies to get a design close to Figma template as much as possible.

> * [ ]  Please make sure to use the same colors for arrows and line after frameworks and skills. See the images below for more clarity.point_down

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/106119105/227723850-d6d4aa66-21ab-42b7-8681-0da4d6e5e149.png)  | ![image](https://user-images.githubusercontent.com/106119105/227723793-20098608-153f-4340-b764-0dc97309cd95.png)

Reply:
- [ ] [OPTIONAL] Well done fixing the incongruencies :hugs: but I'd like to ask if you could make a little tilting of the arrows slightly to the edge rightward (toward the right hand side) as indicated in the screenshot below:
![image](https://user-images.githubusercontent.com/99715640/227750643-ab30011f-9552-4d33-9a9e-5a5267cc6b64.png)